app/lib/config/config.go

```diff
@@ -237,3 +240,13 @@ func (s *ServerToml) String() string {
 	}
 	return buff.String()
 }
+
+// TildeToHome takes a path and replaces an eventual "~" with the home-directory
+func TildeToHome(path string) string {
```

Comment:
> The second link is about using ~ in shell-commands, here I want to use it in os.Open or others, where I cannot do what is written in the accepted solution.

The second [solution](http://stackoverflow.com/a/17617721) is applicable here to me it seems .. ?